### PR TITLE
🎉 Destination databricks: update fields specifications

### LINKED ISSUE #8948: Update fields in destination-connectors specifications: databricks
Add title and description properties to each field in destination-connectors specifications:
destination-databricks

[EPIC] Add title and description properties to each field in connectors specifications and order input fields appropriately #8324

### PR DESCRIPTION
**Pull request by igrankova**
## What
Add title and description properties to each field in destination-connectors specifications:
destination-databricks

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others. 

<details><summary> <strong> Updating a connector </strong></summary>
<p>
   
#### Community member or Airbyter
   
- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
   
#### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items. 
   
- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] Credentials added to Github CI. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci). 
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing. 
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the new connector version is published, connector version bumped in the seed directory as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</p>
</details>

### CODE REVIEW

**Comment by mkhokh-33** [comment]
hi @sherifnada , @tuliren 
Could you please help here 
I see from https://github.com/airbytehq/airbyte/pull/5998
that "This connector needs to be published as a private image."
Could you please help to publish image here.
Tnx

**Review by mkhokh-33 — APPROVED** [review]

**Comment by tuliren** [comment]
> hi @sherifnada , @tuliren Could you please help here I see from #5998 that "This connector needs to be published as a private image." Could you please help to publish image here. Tnx

Once https://github.com/airbytehq/airbyte/pull/7622 is merged, we can publish the image with the `publish` command. That PR was paused for a while, but I will pick it up and finish it, hopefully today.

**Comment by tuliren** [comment]
#7622 is delayed by https://github.com/airbytehq/airbyte/issues/9298, and the integration test cannot pass. I am still working on it.

File: docs/integrations/destinations/databricks.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ Under the hood, an Airbyte data stream in Json schema is first converted to an A
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+| 0.1.3 | 2021-12-28 | [\#9153](https://github.com/airbytehq/airbyte/issues/9153) | Update connector fields title/description |
```
Comment (tuliren) [inline docs/integrations/destinations/databricks.md:106]: I'd like to published a previous change together with this PR:

```suggestion
| 0.1.3 | 2022-01-06 | [\#7622](https://github.com/airbytehq/airbyte/pull/7622) [\#9153](https://github.com/airbytehq/airbyte/issues/9153) | Upgrade Spark JDBC driver to `2.6.21` to patch Log4j vulnerability; update connector fields title/description. |
```

**Comment by tuliren** [comment]
Databricks connector is still private, but it can be tested and published as every other connector in PR directly now.

**Comment by tuliren** [comment]
The new version has been published. This connector is currently not publicly available. So the seed does not need to be updated.

**Pull request merged**